release/models/network-instance/openconfig-network-instance-l2.yang

```diff
@@ -738,11 +755,115 @@ submodule openconfig-network-instance-l2 {
             type oc-evpn-types:evi-id;
             description "Next hop label representing the l2vni for the route";
           }
+          leaf esi {
+            type oc-evpn-types:esi;
+            description "Ethernet Segment Identifier (ESI)";
+            reference "RFC7432: BGP MPLS-Based Ethernet VPN";
+          }
+          leaf resolved {
+            type boolean;
```

Comment:
Is this boolean flag indicates the reachability of a given ESI?

Reply:
This leaf indicates if the path is eligible for forwarding or not.